package.json

```diff
@@ -34,6 +34,7 @@
   "devDependencies": {
     "babel-cli": "^6.10.1",
     "babel-eslint": "^6.0.5",
+    "babel-plugin-transform-regenerator": "^6.21.0",
```

Comment:
I don't see how that's possible seeing how it's not in master.
https://github.com/OpenBazaar/openbazaar-desktop/blob/master/package.json